spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCherrypickSnapshotProcedure.java

```diff
@@ -171,18 +167,10 @@ public void testInvalidCherrypickSnapshotCases() {
 
     AssertHelpers.assertThrows("Should reject calls without all required args",
         AnalysisException.class, "Missing required parameters",
-        () -> sql("CALL %s.system.cherrypick_snapshot('n', 't')", catalogName));
+        () -> sql("CALL %s.system.cherrypick_snapshot('t')", catalogName));
 
     AssertHelpers.assertThrows("Should reject calls with invalid arg types",
         AnalysisException.class, "Wrong arg type for snapshot_id: cannot cast",
-        () -> sql("CALL %s.system.cherrypick_snapshot('n', 't', 2.2)", catalogName));
-
-    AssertHelpers.assertThrows("Should reject empty namespace",
-        IllegalArgumentException.class, "Namespace cannot be empty",
-        () -> sql("CALL %s.system.cherrypick_snapshot('', 't', 1L)", catalogName));
-
-    AssertHelpers.assertThrows("Should reject empty table name",
```

Comment:
Should this have a test for empty table name?

Reply:
+1

Reply:
added